[2024-06-07T12:27:27.778+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-07T12:27:27.804+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xdag_id_1.gett_current_user manual__2024-06-07T12:27:19.873056+00:00 [queued]>
[2024-06-07T12:27:27.816+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xdag_id_1.gett_current_user manual__2024-06-07T12:27:19.873056+00:00 [queued]>
[2024-06-07T12:27:27.817+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-07T12:27:27.833+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): gett_current_user> on 2024-06-07 12:27:19.873056+00:00
[2024-06-07T12:27:27.846+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=15011) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-07T12:27:27.849+0000] {standard_task_runner.py:63} INFO - Started process 15013 to run task
[2024-06-07T12:27:27.849+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xdag_id_1', 'gett_current_user', 'manual__2024-06-07T12:27:19.873056+00:00', '--job-id', '605', '--raw', '--subdir', 'DAGS_FOLDER/jobs_dag.py', '--cfg-path', '/tmp/tmpd3wa2c9c']
[2024-06-07T12:27:27.852+0000] {standard_task_runner.py:91} INFO - Job 605: Subtask gett_current_user
[2024-06-07T12:27:27.910+0000] {task_command.py:426} INFO - Running <TaskInstance: xdag_id_1.gett_current_user manual__2024-06-07T12:27:19.873056+00:00 [running]> on host db8c7b709f57
[2024-06-07T12:27:28.038+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xdag_id_1' AIRFLOW_CTX_TASK_ID='gett_current_user' AIRFLOW_CTX_EXECUTION_DATE='2024-06-07T12:27:19.873056+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-07T12:27:19.873056+00:00'
[2024-06-07T12:27:28.043+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-07T12:27:28.046+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-07T12:27:28.048+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'whoami']
[2024-06-07T12:27:28.065+0000] {subprocess.py:86} INFO - Output:
[2024-06-07T12:27:28.071+0000] {subprocess.py:93} INFO - default
[2024-06-07T12:27:28.072+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-07T12:27:28.074+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-07T12:27:28.113+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=xdag_id_1, task_id=gett_current_user, run_id=manual__2024-06-07T12:27:19.873056+00:00, execution_date=20240607T122719, start_date=20240607T122727, end_date=20240607T122728
[2024-06-07T12:27:28.152+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-07T12:27:28.186+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-07T12:27:28.188+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
